docs/src/main/paradox/java/http/routing-dsl/rejections.md

```diff
@@ -1,137 +0,0 @@
-# Rejections
```

Comment:
But that will contain Scala tests/examples and not contain the Java ones?
We need to update the scala file to include the java examples before this can be merged